packages/yasgui/src/Tab.ts

```diff
@@ -195,19 +194,18 @@ export class Tab extends EventEmitter {
 
   private checkEndpointForCors(endpoint: string) {
     if (this.yasgui.config.corsProxy && !(endpoint in Yasgui.corsEnabled)) {
-      superagent
-        .get(endpoint)
-        .query({ query: "ASK {?x ?y ?z}" })
-        .then(
-          () => {
+      fetch(`${endpoint}?${new URLSearchParams({ query: "ASK {?x ?y ?z}" }).toString()}`)
+        .then((response) => {
+          if (response.ok) {
```

Comment:
This is not the same behavior as the current solution, as `response.ok` can be false in case of a non 2xx status code (see: https://developer.mozilla.org/en-US/docs/Web/API/Response/ok).